lms/djangoapps/certificates/docs/decisions/007-transfer-certificate-signatures-from-mongo.rst

```diff
@@ -0,0 +1,104 @@
+
+Transfer certificate signatures from Mongo to Credentials
+#########################################################
+
+Status
+******
+Proposed
+
+Context
+*******
+As a part of `Old Mongo Deprecation work <https://github.com/openedx/public-engineering/issues/62>`_
+we are planning to remove the support for c4x assets. After removing support for c4x assets in
+StaticContentServer middleware, the certificate signatories for courses with deprecated IDs won't
+be available because those signatures are stored in course assets.
+
+
+Decision
+********
+To keep access to the certificate signatures will be used external signatures from credentials on WEB/PDF
+certificate rendering.
+
+* credentials API must be extended to provide endpoints for adding and retrieving signatures
+
+  * `CourseCertificateViewSet` API must be updated with possibility uploading signature
```

Comment:
by "possibility", do you mean "optionally"?

Reply:
That's right, the signature create/update should be optional when pushing a certificate configuration to Credentials from LMS

Reply:
Updated